### DISCUSSION TITLE
Question about storage utilization

### DISCUSSION THREAD

**Discussion by nordowl** [planetscale/discussion#409] [Q&A]
Hey all,
I was just testing around adding and deleting data. I noticed whenever I add data the storage utilization graph goes up - obviously. But when I delete data, the graph does not go down again.
Is this intended? I'm a bit confused. Is there maybe a cache or junk data I can clear?

Thanks!

**Comment by fooforge** [comment]
Hey @nordowl,

The graphs in the web-interface do not update in realtime, but only every hour, so that might be the reason why you don't see an immediate drop in usage. If you have already factored that in, I'd appreciate if you could open a ticket with us at https://support.planetscale.com/ and send us a screenshot of what you see. I'll then take a look. Thanks!

Schoene Gruesse aus Ostholstein! ;-)
